runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -546,12 +546,27 @@ J9::AheadOfTimeCompile::initializeCommonAOTRelocationHeader(TR::IteratedExternal
          if (aconstNode->getOpCodeValue() == TR::loadaddr)
             j9method = reinterpret_cast<TR_OpaqueMethodBlock *>(aconstNode->getSymbolReference()->getSymbol()->castToStaticSymbol()->getStaticAddress());
 
-         TR_OpaqueClassBlock *j9class = fej9->getClassFromMethodBlock(j9method);
+         TR_InlinedCallSite & ics = comp->getInlinedCallSite(inlinedSiteIndex);
```

Comment:
Please improve the commit summary "MethodPointer Relo Change" isn't the most generic you could have made it, but I think you could do better.

Reply:
Addressed in the latest force push.